### PR TITLE
global: fail fast on retry errors

### PR DESCRIPTION
**Pull request by klaskosk**
As originally used by the cnf/ran directory, retry is only supposed to retry on errors that come from the command execution failing, often from the MCO daemon restarting mid-execution. The current implementation will ignore errors completely that are not to be retried. This masks any errors in the actual command failing.

This PR just changes back to the original behavior and will fail fast when the command gets executed but fails, for example, by having a nonzero exit code.

### CODE REVIEW

**Review by edcdavid — APPROVED** [review]
lgtm, matches previous function now: 
- retries if error is "error executing command in container" or "container not found"
- fail if any other error 
- completes successfully if no error

**Review by kononovn — APPROVED** [review]
/lgtm

File: tests/internal/cluster/cluster.go [hunk 1]
```diff
@@ -193,13 +193,14 @@ func ExecCmdWithRetries(client *clients.Settings, retries uint,
...
 			err := ExecCmd(client, nodeSelector, command)
-
 			if isErrorExecuting(err) {
-				glog.V(90).Infof("Error during command execution, retry %d (%d max): %w", retry, retries, err)
+				glog.V(90).Infof("Error during command execution, retry %d (%d max): %v", retry, retries, err)
 
 				retry++
 
 				return false, nil
+			} else if err != nil {
```
Comment (trewest) [inline tests/internal/cluster/cluster.go:202]: Why else if ?

File: tests/internal/cluster/cluster.go [hunk 2]
```diff
@@ -215,25 +216,29 @@ func ExecCmdWithStdoutWithRetries(
...
-
 			outputs, err = ExecCmdWithStdout(client, command, options...)
 			if isErrorExecuting(err) {
-				glog.V(90).Infof("Error during command execution, retry %d (%d max): %w", retry, retries, err)
+				glog.V(90).Infof("Error during command execution, retry %d (%d max): %v", retry, retries, err)
 
 				retry++
 
 				return false, nil
+			} else if err != nil {
```
Comment (trewest) [inline tests/internal/cluster/cluster.go:234]: same here

File: tests/internal/cluster/cluster.go (hunk 1 above)
Comment (klaskosk) [reply tests/internal/cluster/cluster.go:202]: Oh this can just be a separate if since the first block always returns, I'll update it

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**